content/product/cloud-delivery-service.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Cloud Delivery Service"
+date: 2018-06-01T15:10:06-07:00
+layout: "cloud-delivery-service"
+draft: true
```

Comment:
We should also do an audit of `draft:` at some point. Are these pages not being published currently? If not, can we remove? If they are we should delete the `draft:` from the front matter.

Reply:
I would, yeah. No sense keeping around what clearly isn't in use!